### PR TITLE
New integration SimpleS3

### LINKED ISSUE #620: [S3] S3Client::getUrl()
This was mentioned in #610. 

If you have a public bucket, you may want to have a link to the object. Should we add `S3Client::getUrl()` with an implementation like the one in #610?

Or should this be in a integration layer?

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix #620

The idea is that we can wrap the S3 client in a "simple S3" layer. I only imagine 4 methods. `getUrl`, `put()`, `get()` and `has()`. All with a minimum number of arguments. 

I put it in an integration not to distract from the pure and autogenerated client.

### CODE REVIEW

File: src/Integration/Aws/SimpleS3/composer.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "async-aws/simple-s3",
+    "type": "library",
+    "description": "A simple S3 client that are easy to work with. ",
+    "keywords": [
+        "aws",
+        "async-aws",
```
Comment (jderusse) [inline src/Integration/Aws/SimpleS3/composer.json:7]: amazon, sdk?

**Comment by jderusse** [comment]
Adding test may worth it

**Comment by Nyholm** [comment]
Thank you for the review. I've updated the PR accordingly. 

I've also added a test to find a bug =)

File: src/Integration/Aws/SimpleS3/composer.json (hunk 1 above)
Comment (Nyholm) [reply src/Integration/Aws/SimpleS3/composer.json:7]: Thank you

File: src/Integration/Aws/SimpleS3/tests/Unit/SimpleS3ClientTest.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class SimpleS3ClientTest extends TestCase
+{
+    public function testGetUrl()
+    {
+        $client = new SimpleS3Client([], new Credentials('id', 'secret'), new MockHttpClient());
+        $url = $client->getUrl('bucket', 'images/file.jpg');
+        self::assertSame('https://s3.amazonaws.com/bucket/images/file.jpg', $url);
+    }
+
+    public function testGetUrlWithNoCredentials()
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply src/Integration/Aws/SimpleS3/tests/Unit/SimpleS3ClientTest.php:22]: The tests helped =)

**Comment by Nyholm** [comment]
I'll merge and continue on docs and big file integration

**Pull request merged**